song/song.go

```diff
@@ -18,106 +18,89 @@
 package song
 
 import (
-	"bytes"
-	"io/ioutil"
+	//"fmt"
```

Comment:
If not needed I think we can remove it

Reply:
Yup. I kept adding & removing it during testing; but we don't need it, and it should go. :-)

Reply:
:+1: